WalletWasabi.Documentation/Guides/DeterministicBuildGuide.md

```diff
@@ -10,8 +10,9 @@ In order to reproduce Wasabi's builds you need Git, Windows 10 and the version o
 
 ```sh
 git clone https://github.com/zkSNACKs/WalletWasabi.git
+cd WalletWasabi
 git checkout {hash of the release} // This works from 1.1.3 release, https://github.com/zkSNACKs/WalletWasabi/releases
```

Comment:
`//` -> `#`

Reply:
Doesn't work:

```
C:\Users\user>cd IntelGraphicsProfiles

C:\Users\user\IntelGraphicsProfiles>cd ..

C:\Users\user>cd IntelGraphicsProfiles foo
The system cannot find the path specified.

C:\Users\user>cd IntelGraphicsProfiles #foo
The system cannot find the path specified.

C:\Users\user>cd IntelGraphicsProfiles # foo
The system cannot find the path specified.
```

Reply:
It works in Powershell and in Bash. I guess that's the best thing you can get.

Reply:
cmd, yeah if in PS it works, then it makes more sense to have that as that's Windows thing too. Bash is irrelevant as deterministic build guide specifies win10.